cosmwasm/enclaves/execute/src/registration/attestation.rs

```diff
@@ -289,6 +292,230 @@ pub fn get_mr_enclave() -> [u8; 32] {
     rsgx_self_report().body.mr_enclave.m
 }
 
+
+#[cfg(feature = "SGX_MODE_HW")]
+pub fn verify_quote_ecdsa(
```

Comment:
Needs some description on the logic that's happening here (and in other complex functions)